kemp-loadmaster-multinic/azuredeploy.parameters.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "balPassword": {
+      "value": "GEN-PASSWORD"
+    },
+    "existingvirtualNetworkResourceGroup": {
```

Comment:
Since CI can't be run on this remove all the parameters that would require the user to supply a value - this will force users to supply a value (that's more likely to be correct)